elixir/apps/domain/lib/domain/accounts/config.ex

```diff
@@ -7,6 +7,20 @@ defmodule Domain.Accounts.Config do
       field :protocol, Ecto.Enum, values: [:ip_port, :dns_over_tls, :dns_over_http]
       field :address, :string
     end
+
+    embeds_one :notifications, Notifications,
+      primary_key: false,
+      on_replace: :update do
+      embeds_one :outdated_gateway, OutdatedGateway, primary_key: false, on_replace: :update do
```

Comment:
We can move this schema out and reuse for all email notifications, then we don't need to have a separate schema and a changeset for each of them:
```suggestion
      embeds_one :outdated_gateway, Domain.Account.Config.Notifications.Email, on_replace: :updated
```

Reply:
👍 done